services/transition/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3.7'
+
+x-transition: &transition
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: transition
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/transition
+
+services:
+  transition-lite:
+    <<: *transition
+    depends_on:
+      - nginx-proxy-app
+      - postgres
+      - redis
+      - transition-worker
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres/transition"
+      TEST_DATABASE_URL: "postgresql://postgres@postgres/transition-test"
+
+  transition-app: &transition-app
+    <<: *transition
+    depends_on:
+      - nginx-proxy-app
+      - postgres
+      - redis
+      - transition-worker
```

Comment:
I wonder if we need this. My (possibly extreme!) approach to this is to only add config that I can make use of. So if you're spinning up transition in your browser, and pages error because of the worker not being there (or something else), then we should add it. Otherwise, we could safely leave it out.

Reply:
If we need it later then we can add it then. Good idea.